subprojects/packagefiles/stduuid/test/meson.build

```diff
@@ -0,0 +1,32 @@
+test_src_files = files('main.cpp', 'test_generators.cpp', 'test_uuid.cpp')
+
+test_inc_dirs = inc_dirs
+test_inc_dirs += include_directories('../catch')
+
+test_flags = flags
+
+
+if host_machine.system() == 'windows'
+    test_flags += [
+        windows_options_prefix + 'EHc',
+        windows_options_prefix + 'Zc:hiddenFriend',
+        windows_options_prefix + 'D_SCL_SECURE_NO_WARNINGS',
+    ]
+elif host_machine.system() == 'darwin'
+    test_flags += ['-fexceptions', '-g', '-Wall']
+else
+    test_flags += ['-fexceptions', '-g', '-Wall']
```

Comment:
You must _never_ add `-g` or `-Wall`. That's what debug and warning levels are for.